script.py

```
@@ -889,7 +895,22 @@ def get_video_info_web():
+        if not thumbnail_url and video_url:
+            try:
+                # Extract video ID and construct thumbnail URL
+                if 'youtu.be/' in video_url:
+                    video_id = video_url.split('youtu.be/')[-1].split('?')[0]
+                elif 'v=' in video_url:
+                    video_id = video_url.split('v=')[-1].split('&')[0]
+                else:
+                    video_id = None
+                if video_id:
+                    thumbnail_url = f"https://img.youtube.com/vi/{video_id}/maxresdefault.jpg"
+            except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)